examples/network-hhmodel/incidence_report.rs

```diff
@@ -0,0 +1,54 @@
+use ixa::context::Context;
+use ixa::error::IxaError;
+use ixa::global_properties::ContextGlobalPropertiesExt;
+use ixa::people::PersonPropertyChangeEvent;
+use ixa::report::ContextReportExt;
+use ixa::ContextPeopleExt;
+use ixa::{create_report_trait, report::Report};
+use std::path::PathBuf;
```

Comment:
For suggestion below re: panic on no directory.

```suggestion
use std::{fs, path::PathBuf};
```